examples/Example/src/new_api/camera/index.tsx

```diff
@@ -0,0 +1,351 @@
+import React, { useState } from 'react';
+import { Alert, StyleSheet, useWindowDimensions } from 'react-native';
+import {
+  GestureMonitor,
+  Gesture,
+  useAnimatedGesture,
+} from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  runOnJS,
+} from 'react-native-reanimated';
+
+const filters = ['red', 'green', 'blue', 'yellow', 'orange', 'cyan'];
+const MAX_VIDEO_DURATION = 60_000;
+const CAPTURE_BUTTON_RADIUS = 50;
+const FILTER_BUTTON_RADIUS = 35;
+
+export default function Home() {
+  const filter = useSharedValue(0);
+  const filterOffset = useSharedValue(0);
+  const zoom = useSharedValue(1);
+  const [scale, setScale] = useState(1);
+  const [selectedFilter, setSelectedFilter] = useState(0);
+  const [isRecording, setIsRecording] = useState(false);
+  const [remainingTime, setRemainingTime] = useState(MAX_VIDEO_DURATION);
+  const [recordingIntervalHandle, setRecordingIntervalHandle] = useState(-1);
+
+  const filtersPanGesture = Gesture.pan()
+    .setOnUpdate((e) => {
+      'worklet';
+      filter.value =
+        filter.value + (filterOffset.value - e.translationX) * 0.01;
```

Comment:
```suggestion
        filter.value + (filterOffset.value - e.translationX) / 100;
```

Reply:
Done in c9c43d8757b585808251376d16f6a0182c2842f3